.github/workflows/verify_changes.yml

```
@@ -25,7 +25,7 @@ jobs:
       - name: Get changed directories
         id: get-changeddir
         env:
-          TYPE: ${{ github.event_name }}
-          REF_NAME: ${{ github.ref_name }}
-          BASE: ${{ github.base_ref }}
+          TYPE: "${GITHUB_EVENT_NAME}"
+          REF_NAME: "${GITHUB_REF_NAME}"
+          BASE: "${GITHUB_BASE_REF}"
         run: ./.github/scripts/verify_changes.sh ${{ env.TYPE }} ${{ env.REF_NAME }} ${{ env.BASE }}
```

@fatima2003 Ah, I had meant something like this:

```suggestion
        run: ./.github/scripts/verify_changes.sh "${{ env.TYPE }}" "${{ env.REF_NAME }}" "${{ env.BASE }}"
```

---

Aah makes sense, it felt wrong wrapping those in quotes :D I also think instead of initializing env variables with the default GitHub env variables, we can just use the default variables... maybe that's also why there's an error